domain/mvc/tokens.go

```diff
@@ -40,4 +40,10 @@ type TokensUsecase interface {
 	RegisterPricingStrategy(source domain.PricingSourceType, strategy domain.PricingSource)
 
 	IsValidChainDenom(chainDenom string) bool
+
+	// Check if the pricing source is a valid one
```

Comment:
nit: I know we have some specs missed above but would still be good to continue working towards maximizing consistency
```suggestion
	// IsValidPricingSource check if the pricing source is a valid one
```